floof.yaml

```diff
@@ -38,8 +38,8 @@ frontend:
       run:
         - npx relay-compiler
```

Comment:
Mhh would it make sense to create a `npm run` scriipt for this as well?

Reply:
I thought about it; also for the Webpack tasks ... but I didn't want to go down that perfectionism hole. x) As long as the command is simple enough (no options) and/or the invocations in the different places are sufficiently different, I think we don't **need** `"scripts"` for them.

That being said, I would not oppose a PR adding more `"scripts"`. :wink: